distribution/pom.xml

```diff
@@ -1,368 +1,373 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I tried several times, but no luck. Git always shows the entire contents replaced although I just added 4 lines. This does not occur for other files (I tested it), just this file. I also checked line endings, still no luck. Sorry, but I don't know how to tell git that only 4 lines changed...
Any idea?